apps/ios/LiveVideo/LiveVideo/Launch/LiveVideoApp.swift

```diff
@@ -25,14 +25,20 @@ struct LiveVideoApp: App {
                 .environmentObject(streamManager)
                 .environmentObject(speakerGridViewModel)
                 .environmentObject(speakerSettingsManager)
-                .environmentObject(raisedHandsStore)
+                .environmentObject(viewersViewModel)
                 .environmentObject(api)
                 .onAppear {
                     let localParticipant = LocalParticipantManager(authManager: authManager)
                     let roomManager = RoomManager()
                     roomManager.configure(localParticipant: localParticipant)
                     let viewerStore = ViewerStore()
-                    let syncManager = SyncManager(raisedHandsStore: raisedHandsStore, viewerStore: viewerStore)
+                    let raisedHandsStore = SyncUsersStore()
+                    let viewersStore = SyncUsersStore()
```

Comment:
Assuming raisedHandsStore and viewersStore doesn't share any participants

Reply:
They do share some participants. `raisedHandsStore` is all users that have their hand raised. `viewersStore` is all viewers regardless if hand is raised or not. 

Some more background on the sync architecture. We really need to use a separate sync map for raised hands. There could be 1k users in the viewers sync map. We can't just add a hand raised attribute to users in the `viewers` map because of pagination. We would have to paginate through all users just to find the users that have their hand raised to display in the UI. And all that pagination doesn't scale and isn't feasible. So because of how sync works we must use a separate map just for raised hands. If sync had a way to subscribe to queries that would have helped but it does not.

Also some kind of related discussion here: https://github.com/twilio/twilio-live-interactive-video/pull/55#discussion_r746805646

Reply:
The link I added above tries to explain why we wouldn't have the server manage these two maps such that there is no overlap in users.